app/src/routes/jobs/index.jsx

```diff
@@ -3,10 +3,10 @@ import SearchBar from "src/components/SearchBar";
 
 export default function Jobs() {
```

Comment:
I'm wondering if we can rename this to index or JobsPage if this is the entry point to the Jobs page. Cause there might be a components called Jobs in the components.

Reply:
We can do this in a separate refactor ticket? Cause the logic would apply to all the routes